### ISSUE TITLE
Items { selected: true } is not working?

### ISSUE THREAD

**Issue by twizzlerz04** [pytesNET/tail.select#65]
Seems `items { selected: true }` is not working?

```
tail.select('form[name="editUserForm"] select[name="role-id"]', {
                        search: false,
                        multiPinSelected: false, 
                        placeholder: "Role",
                        items: {
                            1: { value: 1, selected: false, disabled: false },
                            2: { value: 2, selected: true, disabled: false }
                        }
                    });

```

when the modal load the 1 is still selected. Why is that?

**Comment by SamBrishes** [comment]
Hellow,

thanks for mention this issue. It seems like a bug at the moment, I'll check that.

Sincerely,
Sam.

**Comment by Calak** [comment]
@twizzlerz04 can you check if #64 fixes that?

**Comment by Calak** [comment]
@twizzlerz04 I updated PR #64, can you try it please?

**Comment by SamBrishes** [comment]
Hellow,

This has been fixed / added in Version 0.5.13. Thanks.

Sincerely,
Sam.

**Comment by twizzlerz04** [comment]
> @twizzlerz04 I updated PR #64, can you try it please?

my bad, i just tried it and its working now :)) thanks
